cmd/scheduled-feed/main.go

```diff
@@ -58,30 +60,39 @@ func (handler *FeedHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 }
 
 func main() {
-	pubURL := os.Getenv("OSSMALWARE_TOPIC_URL")
-	var pub publisher.Publisher
+	configPath, useConfig := os.LookupEnv("PACKAGE_FEEDS_CONFIG_PATH")
 	var err error
-	if pubURL == "" {
-		pub = publisher.NewStdoutPublisher()
+
+	var appConfig *config.ScheduledFeedConfig
+	if useConfig {
+		appConfig, err = config.FromFile(configPath)
 	} else {
-		pub, err = publisher.NewPubSub(context.TODO(), pubURL)
-		if err != nil {
-			log.Fatalf("error creating gcp pubsub topic with url %q: %v", pubURL, err)
-		}
+		appConfig = config.Default()
```

Comment:
I might be pathing through this wrong, but if no config file is provided, then the env vars for `OSSMALWARE_TOPIC_URL/PORT` are not considered?

Reply:
Fixed.